CodeGen/Generators/NanoFrameworkGenerator.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlTypes;
+using System.IO;
+using System.Text;
+using CodeGen.Generators.NanoFrameworkGen;
+using CodeGen.JsonTypes;
+using Serilog;
+
+namespace CodeGen.Generators
+{
+    /// <summary>
+    /// Code generator for nanoFramework
+    /// Will generate 1 nanoFramework project per unit, a common property folder and a common package file
+    /// </summary>
+    internal static class NanoFrameworkGenerator
+    {
+        private const int AlignPad = 35;
+
+        /// <summary>
+        /// Create the root folder NanoFramewok
+        /// Create all the quantities unit and quantities file
+        /// Create all individual nanoFramework projects
+        /// Create common package file
+        /// Create common properties file
+        /// </summary>
+        /// <param name="rootDir">The root directory</param>
+        /// <param name="quantities">The quantities to create</param>
+        public static void Generate(string rootDir, Quantity[] quantities)
+        {
+            var outputDir = Path.Combine(rootDir, "NanoFramework", "GeneratedCode");
```

Comment:
I would change the directory name to `UnitsNet.NanoFramework` to be consistent with WRC and other projects.